src/components/resources/Resources.tsx

```diff
@@ -12,19 +12,22 @@ import Title from "components/Title/Title";
 import Filter from "components/filter/Filter";
 import Text from "components/Text/Text";
 import { useFilters } from "context/ResourceFilterContext";
-import { useFilter } from "hooks/useFilter";
 import { useTranslation } from "react-i18next";
+import { useResourceFilter } from "hooks/useResourceFilter";
 
 import styles from "./Resources.module.css";
+import SearchBar from "../searchBar/SearchBar";
```

Comment:
Again, this is personal preference but let's have this import in the second category of imports, and not below the CSS module.